extensions/PhabBugz/lib/Feed.pm

```diff
@@ -347,21 +360,148 @@ sub process_revision_change {
 
     Bugzilla->set_user($old_user);
 
-    $self->logger->info("SUCCESS");
+    $self->logger->info('SUCCESS: Revision D' . $revision->id . ' processed');
 }
 
-sub feed_transactions {
-    my ($self, $after) = @_;
+sub process_new_user {
+    my ( $self, $object_phid ) = @_;
+
+    my $phab_user = Bugzilla::Extension::PhabBugz::User->new_from_query( { phids => [ $object_phid ] } );
```

Comment:
`User->new_from_query` is requesting Phabricator to get user's data. Can't we use the data already requested in the `new_users` method?

Reply:
I can try that and see. If we pass in the user data to new() then it should convert it to an object the same as if we use new_from_query().